src/components/Login/index.js

```diff
@@ -1,67 +1,39 @@
-import React, { useEffect, useState } from 'react';
-import { func, shape } from 'prop-types';
+import React, { useState } from 'react';
+import { func } from 'prop-types';
 import {
   StyleSheet,
   Text,
   View,
   TouchableOpacity,
   ActivityIndicator,
-  Alert,
 } from 'react-native';
 import { connect } from 'react-redux';
 
-import { setPatientData } from '../../features/patient/patientDataSlice';
-import { setAuth, clearAuth } from '../../features/auth/authSlice';
+import { setAuth } from '../../features/auth/authSlice';
 import Colors from '../../constants/Colors';
 import {
-  authAsync, buildFhirIssUrl, initializeFhirClient, getBundle,
+  authAsync, buildFhirIssUrl,
 } from '../../resources/fhirAuth';
 
 import PatientPicker, { DEFAULT_PATIENT_ID } from './PatientPicker';
 import SkipLoginButton from './SkipLoginButton';
 
 const Login = ({
-  authResult,
-  clearAuthAction,
-  patientData,
   setAuthAction,
-  setPatientDataAction,
 }) => {
   const [loading, setLoading] = useState(false);
   const [mockPatientId, setPatientId] = useState(DEFAULT_PATIENT_ID);
   const fhirIss = buildFhirIssUrl(mockPatientId);
 
-  useEffect(() => {
-    if (authResult && !patientData) {
-      const {
-        accessToken,
-        additionalParameters: { patient: patientId },
-      } = authResult;
-      const fhirClient = initializeFhirClient(fhirIss, accessToken);
-
-      const queryPatient = async (patientIdProp, fhirClientProp) => {
-        try {
-          const bundle = await getBundle(patientIdProp, fhirClientProp);
-          setPatientDataAction(bundle);
-        } catch (error) {
-          clearAuthAction();
-          setLoading(false);
-          console.error('Error fetching patient data:', error); // eslint-disable-line no-console
-          Alert.alert('Login Error', 'Error fetching patient data.', ['ok']);
-        }
-      };
-
-      queryPatient(patientId, fhirClient);
-    }
-  }, [authResult, patientData]);
-
   const handleLogin = async () => {
     setLoading(true);
     const authResponse = await authAsync(fhirIss);
     if (authResponse) {
-      setAuthAction(authResponse);
-    } else {
       setLoading(false);
```

Comment:
if we dont get `authResponse` shouldn't we also set `setLoading(false)`?

Reply:
last few commits that were just pushed should fix this.